src/V8Platform.h

```diff
@@ -28,52 +28,37 @@ class ETWTracingController : public v8::TracingController {
   ETWTracingController(bool enabled) : enabled_(enabled) {}
   ~ETWTracingController() = default;
 
-  const uint8_t *GetCategoryGroupEnabled(const char *category_group) override;
+  const uint8_t* GetCategoryGroupEnabled(const char* category_group) override;

```

Comment:
>const char*  [](start = 41, length = 12)

Our clang format settings put the star on the right. What tool do you use to format code?